README.md

```diff
@@ -1,7 +1,100 @@
-Generate a compatibility matrix for plugins against Jenkins core.
+Plugin Compatibility Tester (PCT)
+------
+
+Generates a compatibility matrix for plugins against Jenkins core.
 
 See https://wiki.jenkins-ci.org/display/JENKINS/Plugin+Compatibility+Tester for background.
 
+## Running PCT 
+
+### Running PCT in Docker
+
+It is recommended to run PCT in the prepared Docker image.
+You can find it [here](hub.docker.com/r/jenkins/pct/).
```

Comment:
`https://hub.docker.com/r/jenkins/pct/` I guess you mean, but this is a 404 (I suppose pending merge of the PR?).

Reply:
Yes, I can deploy the image there if it helps with review

Reply:
Containers are now available